pkg/apis/khjob/v1/doc.go

```diff
@@ -1,5 +1,5 @@
 // +k8s:deepcopy-gen=package
 // +k8s:defaulter-gen=TypeMeta
-// +groupName=khjob
+// +groupName=comcast.github.io
```

Comment:
did we still want our apigroup to be `comcast.github.io` or do we want to change it to `kuberhealthy.io`

Reply:
this is a breaking change so we'll have to save it for v3...